spec/controllers/document_list_export_request_controller_spec.rb

```diff
@@ -4,36 +4,27 @@
   before do
     log_in_as_gds_editor
 
-    Fog.mock!
-    ENV["AWS_REGION"] = "eu-west-1"
-    ENV["AWS_ACCESS_KEY_ID"] = "test"
-    ENV["AWS_SECRET_ACCESS_KEY"] = "test"
-    ENV["AWS_S3_BUCKET_NAME"] = "test-bucket"
+    @stubbed_client = Aws::S3::Client.new(stub_responses: true)
```

Comment:
optional: might be somewhat more robust to use rspec's [let](https://rspec.info/features/3-12/rspec-core/helper-methods/let/) rather than an instance var for this sort of thing. (I'm still new-ish to rspec though so feel free to disregard if you know better than I do!)

Reply:
No you're absolutely right - good catch!